app/bundles/CoreBundle/Test/MauticMysqlTestCase.php

```diff
@@ -127,12 +127,24 @@ private function applySqlFromFile($file)
     {
         $connection = $this->connection;
         $password   = ($connection->getPassword()) ? " -p{$connection->getPassword()}" : '';
-        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()}$password {$connection->getDatabase()} < {$file} 2>&1 | grep -v \"Using a password\" || true";
+        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()}$password {$connection->getDatabase()} < {$file}";
 
-        $lastLine = system($command, $status);
+        // 0 -> stdin, 1 -> stdout, 2 -> stderr
+        // We only need stderr, in case we need to output errors when the result code != 0.
+        $descriptors = [
+            2 => ['pipe', 'w'],
+        ];
+        $pipes       = [];
+
+        $process = proc_open($command, $descriptors, $pipes);
```

Comment:
Here you may add an env variable MYSQL_PWD, otherwise the console is swarmed with `mysql: [Warning] Using a password on the command line interface can be insecure.`. Don't forget to remove use of `$password` variable. Same cane be done here: https://github.com/mautic/mautic/pull/10759/commits/baf3cbe5d623ed2ae037ea18cb1b777a568b0680#diff-f59000212d948d3f4da0c7ecb818b4a207f104ebce2e714652eb6007cdc57ee5R230
```php
$env = ['MYSQL_PWD' => $connection->getPassword()];

$process = proc_open($command, $descriptors, $pipes);
```

Reply:
jup, good point, I addressed both cases